Agreements/Internal/C_E.txt

```diff
@@ -1,3 +1,3 @@
-COPX:EGBB:*:SIVDA:*:*:London AC Daventry:London AC Clacton:21000:*:SIVDA
```

Comment:
Agreed levels diagram disagrees again (AL to be updated?)

Reply:
Nope, vMATS needs updating. Level by SIVDA is correct.